packages/demo/examples/08-network/index.jsx

```diff
@@ -0,0 +1,141 @@
+/* eslint react/prop-types: 0 */
+import React from 'react';
+
+import {
+  Network,
+  withScreenSize,
+} from '@data-ui/network';
+
+import {
+  defaultGraph
+} from './data';
+
+import ExpandableNetwork from './ExpandableNetwork';
+import GraphWithCustimizedRenderer from './GraphWithCustimizedRenderer';
+
+function renderTooltip({ data }) {
+  const { x, y, id } = data;
+  return (
+    <div>
+      {id &&
+        <div>
+          <strong>{id}</strong>
+        </div>}
+      <div>
+        <strong> x </strong>
+        {x && x.toFixed ? x.toFixed(2) : x}
+      </div>
+      <div>
+        <strong>y </strong>
+        {y && y.toFixed ? y.toFixed(2) : y}
+      </div>
+    </div>
+  );
+}
+
+const ResponsiveNetowork = withScreenSize(({ screenWidth, children, ...rest }) => (
+  <Network
+    width={Math.min(1000, screenWidth / 1.3)}
```

Comment:
since all of these have the same logic for sizes you could factor into a variables or functions?